### PR TITLE
Update gsq-roles.ttl

### PR DESCRIPTION
**Pull request by KellyVance**
syntax fixes, and addition of collection

@DavidCrosswellGSQ  Tech
@UbuntuScott  Content

### CODE REVIEW

File: vocabularies/gsq-roles.ttl [hunk 1]
```diff
@@ -35,22 +35,23 @@
     sdo:name "Geological Survey of Queensland" ;
     sdo:url <https://www.business.qld.gov.au/industries/mining-energy-water/resources/geoscience-information/gsq> .
 
-des-im:administrator
+gsqr:administrator a skos:Concept ;
   a skos:Concept ;
```
Comment (DavidCrosswellGSQ) [inline vocabularies/gsq-roles.ttl:39]: Please remove duplicate

File: vocabularies/gsq-roles.ttl [hunk 2]
```diff
@@ -35,22 +35,23 @@
...
   a dcat:Role ;
   rdfs:label "Data Administrator" ;
   skos:broader des-im:manager ;
   skos:definition "Design, implementation and maintenance of the technology environment that supports/enables the information management lifecycle."@en ;
-  skos:inScheme des-im:DES-IM-Roles ;
+  skos:inScheme <http://linked.data.gov.au/def/gsq-roles> ;
```
Comment (DavidCrosswellGSQ) [inline vocabularies/gsq-roles.ttl:44]: I like this @KellyVance

File: vocabularies/gsq-roles.ttl [hunk 3]
```diff
@@ -35,22 +35,23 @@
...
   skos:prefLabel "Data Administrator" .
 
-gsqr:manager
+gsqr:manager a skos:Concept ;
   a skos:Concept ;
```
Comment (DavidCrosswellGSQ) [inline vocabularies/gsq-roles.ttl:48]: duplicate here

**Review by DavidCrosswellGSQ — CHANGES_REQUESTED** [review]

File: vocabularies/gsq-roles.ttl [hunk 4]
```diff
@@ -159,3 +160,16 @@ gsqr:QdexReportsRoles a skos:Collection ;
...
+    skos:member gsqr:administrator,
+        gsqr:manager,
+        gsqr:contractor,
+        gsqr:driller,
+        gsqr:operator,
+        iso11179:Submitter,
+        isorole:author,
+        isorole:owner,
+        isorole:publisher ;
+    skos:prefLabel "Department of Environment and Science Roles"@en .
```
Comment (UbuntuScott) [inline vocabularies/gsq-roles.ttl:175]: Great idea, removing the non des roles

File: vocabularies/gsq-roles.ttl [hunk 5]
```diff
@@ -159,3 +160,16 @@ gsqr:QdexReportsRoles a skos:Collection ;
         isorole:owner,
         isorole:publisher ;
     skos:prefLabel "Georesources Reports Roles"@en .
+
+gsqr:des-roles a skos:Collection ;
+    skos:definition "A collection of roles used by DES"@en ;
+    skos:member gsqr:administrator,
+        gsqr:manager,
+        gsqr:contractor,
+        gsqr:driller,
```
Comment (UbuntuScott) [inline vocabularies/gsq-roles.ttl:169]: remove

File: vocabularies/gsq-roles.ttl [hunk 6]
```diff
@@ -159,3 +160,16 @@ gsqr:QdexReportsRoles a skos:Collection ;
...
+        gsqr:operator,
```
Comment (UbuntuScott) [inline vocabularies/gsq-roles.ttl:170]: ```suggestion```

Comment (UbuntuScott) [inline vocabularies/gsq-roles.ttl:170]:
```suggestion
```

**Review by UbuntuScott — CHANGES_REQUESTED** [review]

**Review by UbuntuScott — APPROVED** [review]

**Review by DavidCrosswellGSQ — APPROVED** [review]
Nice 👍

**Pull request merged**